todo/src/reducers/index.js

```diff
@@ -0,0 +1,38 @@
+import { MARK_DONE, ADD_TODO } from '../actions/index';
+
+const initialState = {
```

Comment:
Good job getting initialState setup. There's some good stuff [here](https://redux.js.org/recipes/structuringreducers/initializingstate#in-depth) about why we initialize the state, and what it allows us to do with our data as a result